### ISSUE TITLE
Datasets with biases and other related work

### ISSUE THREAD

**Issue by boknilev** [azpoliak/robust-nli#4]
A running issue to keep track of other tasks and datasets with biases, which may be amenable to a similar methodology. 

- [Stance Detection Benchmark: How Robust Is Your Stance Detection?](https://arxiv.org/pdf/2001.01565.pdf)

Other relevant papers:
- [Adversarial Filters of Dataset Biases](https://arxiv.org/abs/2002.04108) - theoretical grounding of adversarial data filtering, experiments on real and synthetic datasets. 
- [HypoNLI: Exploring the Artificial Patterns of Hypothesis-only Bias in Natural Language Inference](https://arxiv.org/pdf/2003.02756.pdf)
- [Syntactic Data Augmentation Increases Robustness to Inference Heuristics](https://arxiv.org/abs/2004.11999)
- [Towards Robustifying NLI Models Against Lexical Dataset Biases](https://arxiv.org/abs/2005.04732)
- [Modular Representation Underlies Systematic Generalization in Neural Natural Language Inference Models](https://arxiv.org/abs/2004.14623)
- [Mind the Trade-off: Debiasing NLU Models without Degrading the In-distribution Performance](https://www.aclweb.org/anthology/2020.acl-main.770.pdf)
- [An Empirical Study on Robustness to Spurious Correlations using Pre-trained Language Models](https://arxiv.org/abs/2007.06778)
- [Towards Debiasing NLU Models from Unknown Biases](https://arxiv.org/pdf/2009.12303.pdf)
- [An Empirical Study on Model-agnostic Debiasing Strategies for Robust Natural Language Inference](https://arxiv.org/pdf/2010.03777.pdf)
- [Gone at Last: Removing the Hypothesis-Only Bias in Natural Language Inference via Ensemble Adversarial Training](https://www.aclweb.org/anthology/2020.emnlp-main.665.pdf)
- [Towards Interpreting and Mitigating Shortcut Learning Behavior of NLU models](https://arxiv.org/abs/2103.06922)
- [Modeling the Second Player in Distributionally Robust Optimization](https://arxiv.org/abs/2103.10282)
- [Predict then Interpolate: A Simple Algorithm to Learn Stable Classifiers](https://arxiv.org/pdf/2105.12628.pdf)

**Comment by boknilev** [comment]
Other related work:
- [Predictive Biases in Natural Language Processing Models: A Conceptual
 Framework and Overview](https://arxiv.org/abs/1912.11078)
- [Undersensitivity in Neural Reading Comprehension](https://arxiv.org/abs/2003.04808)
- [Counterfactual Samples Synthesizing for Robust Visual Question Answering](https://arxiv.org/pdf/2003.06576.pdf)
- [Learning What Makes a Difference from Counterfactual Examples and Gradient Supervision](https://arxiv.org/pdf/2004.09034.pdf)
- [Benchmarking Robustness of Machine Reading Comprehension Models](https://arxiv.org/abs/2004.14004)
- [Shortcut Learning in Deep Neural Networks](https://arxiv.org/abs/2004.07780)
- [Language (Technology) is Power: A Critical Survey of "Bias" in NLP](https://arxiv.org/abs/2005.14050)
- [An Empirical Study on Robustness to Spurious Correlations using Pre-trained Language Models](https://arxiv.org/abs/2007.06778)
- [Intrinsic Bias Metrics Do Not Correlate with Application Bias](https://arxiv.org/abs/2012.15859)
- [Challenges in Automated Debiasing for Toxic Language Detection](https://arxiv.org/pdf/2102.00086.pdf)
- [An Investigation of Critical Issues in Bias Mitigation Techniques](https://arxiv.org/pdf/2104.00170.pdf)
- [Evading the Simplicity Bias: Training a Diverse Set of Models Discovers Solutions with Superior OOD Generalization](https://arxiv.org/pdf/2105.05612.pdf)
